tests/regression/echotest/children-warnings-tests-expected.echo

```diff
@@ -1,6 +1,5 @@
 DEPRECATED: child() will be removed in future releases. Use children() instead.
-WARNING: Negative child index (-1) not allowed in file children-warnings-tests.scad, line 1
-WARNING: Child index (5) out of bounds (1 children) in file children-warnings-tests.scad, line 4
-WARNING: Child index (5) out of bounds (2 children) in file children-warnings-tests.scad, line 4
+WARNING: Children index (5) out of bounds (1 children)
```

Comment:
Looks like this lost the file/line-number info. Can we get that back?

Reply:
Yes. It was previously present on child(), but not on children(); those code paths got merged, and were based on the children() path. But it could easily be added back.